src/azure-cli/azure/cli/command_modules/redis/custom.py

```diff
@@ -281,4 +284,19 @@ def build_identity(mi_system_assigned, mi_user_assigned):
         type=identityType.value,
         user_assigned_identities=userIdentities)
 
+
+def cli_redis_access_policy_create(client, resource_group_name, cache_name, access_policy_name, permissions):
+    from azure.mgmt.redis.models import RedisCacheAccessPolicy
+    param = RedisCacheAccessPolicy(permissions=permissions)
+    return client.begin_create_update(resource_group_name, cache_name, access_policy_name, param)
```

Comment:
```suggestion
    return client.begin_create_update(resource_group_name, cache_name, access_policy_name, param).result()
```
Try this for long running operation

Reply:
@evelyn-ys I had already tried that, but the issue lies with this section of the test

```
        self.cmd('az redis update -n {name} -g {rg} --set redisConfiguration.aadEnabled=false')
        time.sleep()
        result = self.cmd('az redis show -n {name} -g {rg}').get_output_in_json()
```